examples/app-python/README.md

```diff
@@ -28,6 +37,41 @@ The sequences supported by the example are described below:
 * `HelloFriend`: accepts parameter, logging "hello, <parameter>!"
 * `LogLocation`: queries the current satellite location and logs it
 
+## Working with I/O interface
+
+# Configuring GPIO in application
+The sample program supports GPIO connected through FTDI interface.
+To configure GPIO, assign right GPIO pin numbers in example_app.py.
+Kindly note that, GPIO pins used in application must be same as pins declared while adding Payload in ACP.
+
+# Configuring UART in application
+Assign right UART port number (e.g. /dev/ttyUSB0 etc.) in example_app.py. Assign expected Baud rate in example_app.py. Default Baud rate is 9600.
+Kindly note that, sample program assumes that, Tx and Rx are connected in loopback mode.
+
+# Building sample application with I/O interface 
+Build the app using the following command:
+
+```
+docker build --platform=linux/amd64 -t satos-payload-example-app-python .
+```
+
+# Running sample application with I/O interface
+ To support I/O interface, docker should be run in privileged mode.
+
+```
+docker run --platform=linux/amd64 -e CONFIG=$CONFIG --privileged -v $(pwd):/workspace -it satos-payload-example-app-python
+```
+
+# Testing I/O interface
+You may now use the Antaris Cloud Platform to submit payload sequences. 
+
+The sequences supported by the example are described below:
+* `Read_FTDI_GPIO` : Reads level (high/low) of GPIO pin connected through FTDI interface.
+* `Write_FTDI_GPIO` : Write <parameter> to GPIO pin connected thrugh FTDI interface. The <parameter> can be `1`(high) or '0' (low)
+* `UART_Loopback` : Sequence to test UART loopback.
+* `Get_IO_Details` : Gives details of interface declared in config.json file 
+* `Get_Intr_Pin` : Gives pin number of Interrupt pin.
```

Comment:
Please use upper camel case instead of upper snake case for these payload sequence names, just so we keep with the current convention. For example, we have a current command called `LogLocation`. it is not `Log_Location` with an underscore. I ask that you rename `Get_IO_Details` to `GetIODetails` to keep aligned.

Reply:
It will be a little awkward to combine Read_FTDI_GPIO to ReadFTDIGPIO so in that case let's just drop FTDI: ReadGPIO

Reply:
Changes are done. In case of GPIO, our current SatOS sdk is supporting only FTDI. Hence, FTDI is mentioned in sequence name. Can I rename it to FTDIReadGPIO and FTDIWriteGPIO?